### PR TITLE
library: add ceph_pool module

### PR DESCRIPTION
**Pull request by guits**
This commit adds a new module `ceph_pool`

Signed-off-by: Guillaume Abrioux <gabrioux@redhat.com>

### CODE REVIEW

File: library/ceph_pool.py [hunk 1]
```diff
@@ -0,0 +1,484 @@
+#!/usr/bin/python3
+# Copyright 2018, Red Hat, Inc.
```
Comment (dsavineau) [inline library/ceph_pool.py:2]: 2020 maggle

**Comment by guits** [comment]
jenkins test centos-non_container-all_daemons

**Comment by guits** [comment]
jenkins test centos-container-all_daemons

**Comment by guits** [comment]
jenkins test centos-non_container-all_daemons

**Comment by guits** [comment]
jenkins test centos-non_container-all_daemons

**Comment by guits** [comment]
jenkins test centos-non_container-all_daemons

**Comment by guits** [comment]
jenkins test centos-non_container-all_daemons

**Comment by guits** [comment]
jenkins test centos-non_container-all_daemons

**Comment by guits** [comment]
jenkins test centos-non_container-all_daemons

**Comment by guits** [comment]
jenkins test centos-non_container-all_daemons

**Comment by guits** [comment]
jenkins test centos-non_container-all_daemons

**Comment by guits** [comment]
jenkins test centos-non_container-all_daemons

**Comment by guits** [comment]
jenkins test centos-non_container-all_daemons

**Comment by guits** [comment]
jenkins test centos-non_container-all_daemons

**Comment by guits** [comment]
jenkins test centos-non_container-all_daemons

**Comment by guits** [comment]
jenkins test centos-non_container-all_daemons

**Comment by guits** [comment]
jenkins test centos-non_container-all_daemons

**Comment by guits** [comment]
jenkins test centos-non_container-all_daemons

**Comment by guits** [comment]
jenkins test centos-non_container-all_daemons

**Comment by guits** [comment]
jenkins test centos-non_container-all_daemons

**Comment by guits** [comment]
jenkins test centos-non_container-all_daemons

**Comment by guits** [comment]
jenkins test centos-container-all_daemons

**Comment by guits** [comment]
jenkins test centos-container-all_daemons

**Comment by guits** [comment]
jenkins test centos-non_container-all_daemons

**Comment by guits** [comment]
jenkins test centos-container-ooo_collocation

**Comment by guits** [comment]
jenkins test centos-non_container-switch_to_containers

**Comment by guits** [comment]
jenkins test centos-container-all_daemons

**Comment by guits** [comment]
jenkins test centos-container-all_daemons

**Comment by guits** [comment]
jenkins test centos-container-purge

**Comment by guits** [comment]
jenkins test centos-container-update

**Comment by guits** [comment]
jenkins test centos-non_container-switch_to_containers

**Comment by guits** [comment]
jenkins test centos-container-update

**Comment by guits** [comment]
unrelated failure in `switch_to_containers` job

**Comment by dsavineau** [comment]
jenkins test centos-non_container-switch_to_containers

**Comment by guits** [comment]
jenkins test centos-non_container-switch_to_containers

**Comment by guits** [comment]
jenkins test centos-non_container-switch_to_containers

**Pull request merged**